inference-engine/tests/ie_test_utils/common_test_utils/ngraph_test_utils.cpp

```diff
@@ -599,6 +512,186 @@ class CompareNodesAttributes {
     attr_comparison::ReadAndStoreAttributes m_store_attr;
     attr_comparison::ReadAndCompareAttributes m_compare_attr;
 };
+
+namespace subgraph_comparison {
+
+struct NodeAndInputDescription {
+    using SubGraphOp = ngraph::op::util::SubGraphOp;
+    using InputDescripton = SubGraphOp::InputDescription;
+    using Parameter = ngraph::opset5::Parameter;
+
+    const ngraph::Input<ngraph::Node> input;
+    const Parameter* parameter;
+    const InputDescripton* description;
+
+    static bool equal_descriptions(const InputDescripton* lhs, const InputDescripton* rhs) {
+        if (!lhs || !rhs || lhs->get_type_info() != rhs->get_type_info()) {
+            return false;
+        }
+
+        if (lhs->get_type_info() == SubGraphOp::SliceInputDescription::type_info) {
+            using InDesc = SubGraphOp::SliceInputDescription;
+            const InDesc* l_input = static_cast<const InDesc*>(lhs);
+            const InDesc* r_input = static_cast<const InDesc*>(rhs);
+            return l_input->m_start == r_input->m_start && l_input->m_stride == r_input->m_stride &&
+                   l_input->m_part_size == r_input->m_part_size &&
+                   l_input->m_end == r_input->m_end && l_input->m_axis == r_input->m_axis;
+        } else if (lhs->get_type_info() == SubGraphOp::MergedInputDescription::type_info) {
+            return true;  // noting extra to check
+        } else if (lhs->get_type_info() == SubGraphOp::InvariantInputDescription::type_info) {
+            return true;  // noting extra to check
+        }
+
+        return false;
```

Comment:
why false?
I suggest to add 
else {
    // exception (unhandled case)
}
return true

Reply:
done